components/table/LoadingRow.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useRef } from 'react'
+import { Table } from '@oacore/design'
+
+const LoadingRow = React.memo(({ pageNumber, observe, unObserve }) => {
+  const componentRef = useRef(null)
+
+  useEffect(() => {
+    observe(componentRef.current)
+    return () => unObserve(componentRef.current)
+  }, [])
+
+  return (
+    <Table.Body ref={componentRef} pagenumber={pageNumber}>
+      <Table.Row>
+        <Table.Cell colSpan={1000}>IsLoading</Table.Cell>
```

Comment:
You could not be able to do so if I did not pass rest props 😉

Reply:
I don't say passing restProps is wrong. I just say it has to be used carefully.